includes/admin/class-settings-manager.php

```
@@ -0,0 +1,508 @@
+                <label><input type="radio" name="cirrusly_shipping_config[profile_mode]" value="single" '.checked('single', $profile_mode, false).' '.esc_attr($disabled_attr).'> Single</label><br>
+                <label><input type="radio" name="cirrusly_shipping_config[profile_mode]" value="multi" '.checked('multi', $profile_mode, false).' '.esc_attr($disabled_attr).'> Mixed</label><br>
+                <div style="display:'.($profile_mode==='multi'?'block':'none').';">
+                    Secondary: <input type="number" step="0.1" name="cirrusly_shipping_config[payment_pct_2]" value="'.esc_attr(isset($config['payment_pct_2'])?$config['payment_pct_2']:3.49).'" style="width:60px"> % + <input type="number" step="0.01" name="cirrusly_shipping_config[payment_flat_2]" value="'.esc_attr(isset($config['payment_flat_2'])?$config['payment_flat_2']:0.49).'" style="width:60px"> $<br>
+                    Split: <input type="number" name="cirrusly_shipping_config[profile_split]" value="'.esc_attr(isset($config['profile_split'])?$config['profile_split']:100).'" style="width:60px"> % Primary
+                </div>
+            </div></div></div>';
+
+        // Matrix
+        echo '<div class="cc-settings-card"><div class="cc-card-header"><h3>3. Scenario Matrix</h3></div><div class="cc-card-body"><table class="widefat striped cc-settings-table"><thead><tr><th>Key</th><th>Label</th><th>Multiplier</th><th></th></tr></thead><tbody id="cc-matrix-rows">';
+        $idx=0;
+        foreach($matrix_rules as $k => $rule) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$k'. (undefined)

(UnusedLocalVariable)